pkg/controllers/v1alpha2/mpi_job_controller.go

```diff
@@ -993,10 +993,10 @@ func newLauncherRole(mpiJob *kubeflow.MPIJob, workerReplicas int32) *rbacv1.Role
 		},
 		Rules: []rbacv1.PolicyRule{
 			{
-				Verbs:         []string{"get"},
-				APIGroups:     []string{""},
-				Resources:     []string{"pods"},
-				ResourceNames: podNames,
+				Verbs:     []string{"get", "list", "watch"},
+				APIGroups: []string{""},
+				Resources: []string{"pods"},
+				//ResourceNames: podNames,
```

Comment:
Why is this commented out?

Reply:
`list` and `watch` action is conflict with `ResourceNames`, I will double check it.

Reply:
Have you double checked on this?

Reply:
yes, it will be conflicted with `list`

Reply:
Then should this be removed instead of commenting out?

Reply:
done.